From: Cordell Bloor <cgmb@debian.org>
Date: Fri, 12 Sep 2025 20:16:09 -0600
Subject: fix missing symbols

The hsaKmtCreateQueueExt and hsaKmtRegisterGraphicsHandleToNodesExt
symbols were added between 6.2 and 6.4 but were not added to the
symbol version script. Upstream doesn't want to maintain libhsakmt
as separate from rocr-runtime, so perhaps they're not maintaining
the infrastructure for shared library builds anymore? It might be
worth considering dropping the libhsakmt1 package entirely.

Forwarded: no
---
 libhsakmt/src/libhsakmt.ver | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libhsakmt/src/libhsakmt.ver b/libhsakmt/src/libhsakmt.ver
index fa2a427..72bb3f7 100644
--- a/libhsakmt/src/libhsakmt.ver
+++ b/libhsakmt/src/libhsakmt.ver
@@ -18,6 +18,7 @@ hsaKmtQueryEventState;
 hsaKmtWaitOnEvent;
 hsaKmtWaitOnMultipleEvents;
 hsaKmtCreateQueue;
+hsaKmtCreateQueueExt;
 hsaKmtUpdateQueue;
 hsaKmtDestroyQueue;
 hsaKmtSetQueueCUMask;
@@ -30,6 +31,7 @@ hsaKmtRegisterMemory;
 hsaKmtRegisterMemoryToNodes;
 hsaKmtRegisterMemoryWithFlags;
 hsaKmtRegisterGraphicsHandleToNodes;
+hsaKmtRegisterGraphicsHandleToNodesExt;
 hsaKmtShareMemory;
 hsaKmtRegisterSharedHandle;
 hsaKmtRegisterSharedHandleToNodes;
